src/components/DualTimeline.tsx

```diff
@@ -0,0 +1,376 @@
+          <Entries>
+            {sortMerge(entries, timelines[1]?.years[index]?.entries ?? []).map(([entryA, entryB], index) => (
+              <Pair key={index}>
+                <Entry right>
+                  {entryA && (
+                    <Data right>
+                      <Text isMajor={entryA?.isMajor}>{entryA.text}</Text>
+                      <When>{entryA.when}</When>
+                    </Data>
+                  )}
+                  <Track forHeading={headings[0]} entry={entryA} />
+                </Entry>
+                <Entry>
+                  <Track forHeading={headings[1]} entry={entryB} />
+                  {entryB && (
+                    <Data>
+                      <Text isMajor={entryB?.isMajor}>{entryB.text}</Text>
+                      <When>{entryB.when}</When>
+                    </Data>
+                  )}
+                </Entry>
+              </Pair>
+            ))}
+          </Entries>
+        </>
+      ))}
+      <LineStartsEnds headings={headings} />
+      <LineStartsEnds ends headings={headings} />
+    </Container>
+  )
+}
+
+const DualTimelineMobile = ({ timelines }: Props) => {
+  const years = timelines.reduce((years, timeline, index) => {
+    timeline.years.forEach(({ year, entries }) => {
+      if (years[year] === undefined) {
+        years[year] = []
+      }
+      years[year][index] = { title: timeline.title, entries }
+    })
+    return years
+  }, {})
+
+  // The track will be 3 'm' length, exactly how "3em" works 🙂
+  const trackTitle = 'mmm'
+
+  return (
+    <Container>
+      <LineStartsEnds isSingle starts headings={[trackTitle]} />
+      {Object.entries(years).map(([year, values]) => (
+        <>
+          <Year value={year} headings={[trackTitle]} isSingle />
+          {values.map(({ title, entries }) => (
+            <>
+              {entries && (
+                <>
+                  <Entries>
+                    <Entry>
+                      <Track forHeading={trackTitle} />
+                      <Heading isSingle>{title}</Heading>
+                    </Entry>
+                  </Entries>
+                  <LineStartsEnds isSingle height="2rem" headings={[trackTitle]} />
+                </>
+              )}
+              {entries?.map((entry, index) => (
+                <Entry isSingle key={index}>
+                  <Track forHeading={[trackTitle]} entry={entry} />
+                  <Data>
+                    <Text isMajor={entry?.isMajor}>{entry.text}</Text>
+                    <When>{entry.when}</When>
+                  </Data>
+                </Entry>
+              ))}
+            </>
+          ))}
+        </>
+      ))}
+      <LineStartsEnds isSingle headings={[trackTitle]} />
+      <LineStartsEnds isSingle ends headings={[trackTitle]} />
+    </Container>
+  )
+}
+
+const Container = styled.div`
+  width: 100%;
+  margin-bottom: 226px;
+`
+
+const Text = styled.div`
+  ${({ isMajor }) =>
+    isMajor &&
+    `
```

Comment:
Although this is correct, using `css` would improve the code highlighting
```suggestion
    css`
```